performant-software/Annotation-Studio
# Anthology Tweaks

## What does this PR do?
1. Added edit button to anthology show only visible to admins or anthology owners
2. Raised banner file size limit to 5 MB
3. Students and Teachers can edit their own anthologies, but not others'.
4. Inserted "(click on title, author, created to sort)" after "Documents" on Anthology show view.
5. Inserted "Search to add documents to your anthology" to the left of the search boxes on Anthology show view.
6. Upon clicking "Assign to Anthology" on a document, the user will be redirected to the same search results that you were on.

## How to test?
1. Go to your own anthology show page editable by yourself, eg (http://cove-staging.herokuapp.com/anthologies/brand-new)
2. There should be an edit button if it is your own anthology.
3. "click on title, author, created to sort", "Search to add documents to your anthology" Text should also be present.
4. When you search for a document and click assign to anthology, it should redirect to the searched documents.
5. Click on the edit button near the title of your own anthology
6. To test the banner, Upload file size larger than 5 mb, it should not work.
7. Upload file smaller than 5 mb and it should work.
